### ISSUE TITLE
Pause button is missing for video header

### ISSUE THREAD

**Issue by westonruter** [Automattic/amp-wp#4594]
## Bug Description

As noted by @swissspidy in #2642:

> In the non-AMP version, `wp-includes/js/wp-custom-header.js` adds a button with the ID `wp-custom-header-video-button` that allows to pause/play the video header, to make custom video headers more accessible.
>
> It would be nice if we could keep this accessibility level. Perhaps in a separate PR?

This PR looks to add that button.

There is one issue, however: `amp-bind` is not set up to allow toggling the `paused` state of an `amp-video`. Neither is there a `togglePlay` action exposed on `amp-video`. There are only `play` and `pause` actions. This means that the singular `wp-custom-header-video-button` will not work as-is in how it toggles pause/play. There will need to be two separate buttons, one for `pause` and another for `play` and only one can be shown at a time. For accessibility, we'll need to ensure that `focus` is put on the alternate button when toggling.

We should also consider adding the play/pause button for the YouTube videos.

## Expected Behaviour

There should be a pause button.

## Steps to reproduce

1. Activate Twenty Seventeen
2. Open Customizer
3. Assign a video header
4. Compare the AMP and non-AMP versions of a page
5. Notice no pause button is on the non-AMP version

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*
